Sources/NIOPostgres/Data/PostgresRow.swift

```diff
@@ -2,50 +2,64 @@ public struct PostgresRow: CustomStringConvertible {
     final class LookupTable {
         let rowDescription: PostgresMessage.RowDescription
         let resultFormat: [PostgresFormatCode]
-        
+
+        struct Entry {
+            let indexInRow: Int
+            let field: PostgresMessage.RowDescription.Field
+        }
+        let columnNameToIndexLookupTable: [String: [Entry]]
+
         init(
             rowDescription: PostgresMessage.RowDescription,
             resultFormat: [PostgresFormatCode]
-        ) {
+            ) {
             self.rowDescription = rowDescription
             self.resultFormat = resultFormat
+
+            var columnNameToIndexLookupTable: [String: [Entry]] = [:]
```

Comment:
I don't think we need to store this as an array since `lookup` just calls `first` anyway. If we ever add support for getting all values for a given column name we could change this. But IIRC Fluent 3 only lets you get the first column anyway, and I think Fluent 4 should follow suit.

Reply:
if `tableOID != 0`, we actually check a different column name:

```
            if tableOID == 0 {
                return columnTable.first
            } else {
                return columnTable.first { $0.field.tableOID == tableOID }
            }
```

I think we should keep this; otherwise fetching joined tables with two identical column names will likely fail.

Reply:
Ah okay, missed that part. I think we should consider removing support for getting a column for a specific tableOID. That can be a separate PR though so I don't slow things down here. Thanks!